x/stake/client/rest/utils.go

```diff
@@ -0,0 +1,229 @@
+package rest
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/wire"
+	"github.com/cosmos/cosmos-sdk/x/stake"
+	"github.com/cosmos/cosmos-sdk/x/stake/tags"
+	"github.com/cosmos/cosmos-sdk/x/stake/types"
+	"github.com/pkg/errors"
+	rpcclient "github.com/tendermint/tendermint/rpc/client"
+)
+
+// contains Checks if the a given query contains one of the tx types
+func contains(stringSlice []string, txType string) bool {
+	for _, word := range stringSlice {
+		if word == txType {
+			return true
+		}
+	}
+	return false
+}
+
+func getDelegatorValidator(cliCtx context.CLIContext, cdc *wire.Codec, delegatorAddr sdk.AccAddress, validatorAccAddr sdk.AccAddress) (
+	validator types.BechValidator, httpStatusCode int, errMsg string, err error) {
+
+	// Check if the delegator is bonded or redelegated to the validator
+	keyDel := stake.GetDelegationKey(delegatorAddr, validatorAccAddr)
+	// keyRed := stake.GetREDsByDelToValDstIndexKey(delegatorAddr, validatorAccAddr)
```

Comment:
why is this commented out?

Reply:
Oh it was an attempt to create another key in order to get incoming redelegations, but Fabo already discovered how to do so without creating it. Will take it out